{"name": "test_case_4", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "@allure.severity(\"blocker\")\n    def test_case_4():\n        \"\"\"修改个人信息-修改自己的个人信息，修改成功\"\"\"\n        print(\"test case 444444\")\n>       assert 1 == 2\nE       assert 1 == 2\n\ntest_01.py:37: AssertionError"}, "description": "修改个人信息-修改自己的个人信息，修改成功", "attachments": [{"name": "stdout", "source": "9df73225-7ae8-4863-822e-eff6f6da5167-attachment.txt", "type": "text/plain"}], "start": 1600681046089, "stop": 1600681046090, "uuid": "e5b94696-75b9-45b1-8b3c-91daa4ce7eb4", "historyId": "3a2adace8680d18c7b7e2bcfda054c08", "testCaseId": "813bbd959c257376d9ec0b47af6ebcde", "fullName": "test_01#test_case_4", "labels": [{"name": "severity", "value": "blocker"}, {"name": "suite", "value": "test_01"}, {"name": "host", "value": "PC-20160818TLFG"}, {"name": "thread", "value": "14308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_01"}]}